tests/test_language_modeling_example.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os
+import re
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from .test_examples import TIME_PERF_FACTOR
+
+
+prev_quant_model_name = None
+prev_quant_rank = 0
+
+if os.environ.get("GAUDI2_CI", "0") == "1":
+    # Gaudi2 CI baselines
+    MODELS_TO_TEST = {
+        "bf16_1x": [
+            ("google/gemma-2b-it", "mamamiya405/finred", "Habana/gpt2", True, 9.5, 31.5),
+            ("google/gemma-2b-it", "mamamiya405/finred", "Habana/gpt2", False, 6.5, 24.01),
+        ],
+    }
+
+def _test_language_modeling(
+    model_name: str,
+    baseline_train: float,
+    baseline_eval: float,
+    token: str,
+    dataset_name: str,
+    gaudi_config_name: str,
+    use_lazy_mode: bool = False,
+    per_device_train_batch_size: int = 16,
+    per_device_eval_batch_size: int = 16,
+    num_train_epochs: int = 1,
+):
+    command = ["python3"]
+    path_to_example_dir = Path(__file__).resolve().parent.parent / "examples"
```

Comment:
Very minor style point, not blocking. 

Have `Path(__file__).resolve().parent.parent` set to something like OPTIMUM_ROOT as a constant, and then set path to example dir from that constant. Just to make it clearer if someone rearranges the directory structure.

Reply:
Actually I just copy-pasted the same structure from the others existent tests. Should I use a different strategy here?

Reply:
Then it's probably not worth it. It would be slightly easier to change in the future but if they have to make the changes anyway probably easier to just have it be the same change every step of the way.